routes/api/contacts.js

```diff
@@ -1,25 +1,132 @@
 const express = require('express')
-
 const router = express.Router()
+const {Contacts} = require ("../schema")
+const { validateJoi } = require('./validation')
+
 
-router.get('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
+router.get('/', async (req, res) => {
+   try{
+    const contacts = await Contacts.find()
+      res.json ({
+      status: "success",
+      code: 200,
+      data: contacts
+    })
+  } catch (error) {
+    res.status(400).json({
+      status: "error",
+      code: 400,
+      message: "Bad request"
+    })
+  }
 })
 
-router.get('/:contactId', async (req, res, next) => {
-  res.json({ message: 'template message' })
+router.get('/:contactId', async (req, res) => {
+  const {contactId} = req.params;
+  try {
+    const contact = await Contacts.findById(contactId)
+    res.json ({
+      status: "success",
+      code: 200,
+      data: contact,
+      message: "Your contact has been found"
+    })
+  }   catch (error) {
+    res.status(400).json({
+      status: "error",
+      code: 400,
+      message: "Bad request"
+    })
+  }
+  
 })
 
-router.post('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
+router.post('/', async (req, res) => {
+  const validate = validateJoi(contact)  
```

Comment:
Tu musisz sprawdzić czy nie ma błędu, i jeśli jest to zwrócić odpowieni status i wiadomość do frontendu i zakończyć operacje.